### PR TITLE
Many alias names

### PR DESCRIPTION
**Pull request by olbjo**
Handle multiple alias names.

### CODE REVIEW

File: src/common/pf_lldp.h [hunk 1]
```diff
@@ -430,6 +430,16 @@ int pf_lldp_recv (
    pnal_buf_t * p_frame_buf,
    uint16_t offset);
 
+/**
+ * Check if an alias name matches any of the LLDP peers.
+ *
+ * @param net              InOut: The p-net stack instance
+ * @param alias            In:    Alias to be compared with LLDP peers
```
Comment (pyhys) [inline src/common/pf_lldp.h:437]: `* @param alias            In:    Alias to be compared with LLDP peers. Null terminated string.`

File: src/common/pf_lldp.c [hunk 2]
```diff
@@ -1881,27 +1914,41 @@ void pf_lldp_update_peer (
...
+      PF_LLDP_LOG,
+      "LLDP(%d): Port %d, peer info changed\n",
+      __LINE__,
+      loc_port_num);
 
-      strncpy (
-         net->cmina_current_dcp_ase.alias_name,
-         new_alias,
-         sizeof (net->cmina_current_dcp_ase.alias_name));
+   LOG_INFO (
```
Comment (pyhys) [inline src/common/pf_lldp.c:1923]: Remember to show the old peer info only if `p_port_data->lldp.is_peer_info_received`

**Review by pyhys — COMMENTED** [review]
Great!
Remember to update the commit message about "exclude alias".

File: src/common/pf_lldp.c (hunk 2 above)
Comment (olbjo) [reply src/common/pf_lldp.c]: Done

File: src/common/pf_lldp.h (hunk 1 above)
Comment (olbjo) [reply src/common/pf_lldp.h]: done

**Review by pyhys — APPROVED** [review]
Great!

File: src/common/pf_dcp.c [hunk 3]
```diff
@@ -492,8 +492,13 @@ static int pf_dcp_get_req (
          value_length = (uint16_t)strlen ((char *)p_value);
          break;
       case PF_DCP_SUB_DEV_PROP_ALIAS:
-         skip = append_alias_name ? 0 : 1;
-         value_length = (uint16_t)strlen ((char *)p_value);
+         skip = alias_name ? 0 : 1;
+         if (skip == 0)
```
Comment (hefloryd) [inline src/common/pf_dcp.c:496]: Skip is a boolean, `skip = alias_name == NULL` and `if (!skip)`?

Comment (olbjo) [reply src/common/pf_dcp.c]: Done

**Pull request merged**